sheets/templates/admin.html

```diff
@@ -22,24 +22,5 @@ <h3 style="color: green">Authentication successful!</h3>
         <input type="submit" value="Authenticate and authorize with Google" />
       </form>
     </fieldset>
-    {% if alternative_sheets_processing %}
```

Comment:
Is `alternative_sheets_processing` obsolete?

Reply:
Correct. The management commands can accomplish everything that was provided under this flag